docs/docs/recipes.md

```diff
@@ -853,6 +853,124 @@ export const pageQuery = graphql`
 - [Guide to creating pages from data programmatically](/docs/programmatically-create-pages-from-data/)
 - [Example repo](https://github.com/gatsbyjs/gatsby/tree/master/examples/recipe-sourcing-markdown) for this recipe
 
+### Sourcing data from Contentful
+
+#### Prerequisites
+
+- A [Contentful account](https://www.contentful.com/)
+- The [Contentful CLI](https://www.npmjs.com/package/contentful-cli) installed
+
+#### Directions
+
+1. Login to Contentful with the CLI. It will help you create an account if you don't have one.
+
+```
+contentful login
+```
+
+2. Create a new space. Make sure to save the space ID given to you at the end of the command.
+
+```
+contentful space create --name 'Gatsby example'
+```
+
+3. Seed the new space with example blog content using the new space ID returned from the previous command.
+
+```
+contentful space seed -s '[space ID]' -t blog
+```
```

Comment:
````suggestion
```

For example, with a space ID in place: `contentful space seed -s '22fzx88spbp7' -t blog`
````
I think it would help to show an example.